app/Domain/Integrations/Controllers/IntegrationController.php

```diff
@@ -290,20 +290,24 @@ public function requestActivation(string $id, RequestActivationRequest $request)
         }
 
         $organization = OrganizationMapper::mapActivationRequest($request);
-        $this->organizationRepository->save($organization);
-
         $organizers = $request->input('organizers') ?? [];
-        foreach ($organizers as $organizer) {
-            $organizer = new Organizer(
-                Uuid::uuid4(),
-                Uuid::fromString($id),
-                Uuid::fromString($organizer['id'])
-            );
 
-            $this->organizerRepository->create($organizer);
-        }
+        DB::transaction(function () use ($organizers, $id, $organization, $request): void {
```

Comment:
This is not the place to put DB logic. If we want to do this in a transaction then we need to implement an extra method on the `OrganizerRepository` that saves a array of organizers.